psygnal/_signal.py

```diff
@@ -471,7 +471,9 @@ def _raise_connection_error(self, slot: Callable, extra: str = "") -> NoReturn:
 
     def _normalize_slot(self, slot: NormedCallback) -> NormedCallback:
         if ismethod(slot):
-            return (weakref.ref(slot.__self__), slot.__name__)  # type: ignore
+            return _get_proper_name(slot)  # type: ignore
```

Comment:
At this place, I do not know if I could do something better than ignore warning.